lowest-resolved-tree.txt

```diff
@@ -0,0 +1,460 @@
+astropy
+├── astropy-iers-data v0.2025.12.1.0.45.12
+├── numpy v1.25.0
+├── packaging v24.1
+├── pyerfa v2.0.1.1
+│   └── numpy v1.25.0
+├── pyyaml v6.0
+├── asdf-astropy v0.7.0 (extra: all)
+│   ├── asdf v2.14.4
+│   │   ├── asdf-standard v1.1.0
+│   │   ├── asdf-transform-schemas v0.5.0
+│   │   │   └── asdf-standard v1.1.0
+│   │   ├── asdf-unit-schemas v0.1.0
+│   │   │   └── asdf-standard v1.1.0
+│   │   ├── importlib-metadata v4.13.0
+│   │   │   └── zipp v0.5.0
+│   │   ├── jmespath v0.7.1
+│   │   ├── jsonschema v4.0.1
+│   │   │   ├── attrs v19.2.0
+│   │   │   └── pyrsistent v0.19.1
+│   │   ├── numpy v1.25.0
+│   │   ├── packaging v24.1
+│   │   ├── pyyaml v6.0
+│   │   └── semantic-version v2.8.0
+│   ├── asdf-coordinates-schemas v0.3.0
+│   │   ├── asdf v2.14.4 (*)
```

Comment:
note to reviewers: the `(*)` denotes a subtree that was deduplicated, indicating that the package already appeared once.